### PR TITLE
Fix #1629 : replace story_text_size to reading_text_size

### LINKED ISSUE #1629: Update `StoryTextSize` to `ReadingTextSize`
Mock Link: https://xd.adobe.com/view/e8aa4198-3940-47f9-514a-f41cc54457f6-9e9b/screen/2776cbb1-22b0-42c9-b70b-2136bdb361da/

Earlier in the app we were using `StoryTextSize` text in `OptionsActivity` as well as `ExplorationPlayer`. Now that text has been updated to `ReadingTextSize`.

This change should be made in all files, test cases, domain layer, proto layer, xml files, etc.

### PR DESCRIPTION
**Pull request by prayutsu**
## Explanation

Fixes #1629 
All occurrences of StoryTextSize/story text size/story_text_size etc have been replaced with ReadingTextSize/reading text size/reading_text_size

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by prayutsu** [comment]
@rt4914 PTAL

File: app/src/main/java/org/oppia/app/options/OptionsReadingTextSizeViewModel.kt [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package org.oppia.app.options
+
+import androidx.databinding.ObservableField
+
+/** ReadingText size settings view model for the recycler view in [OptionsFragment]. */
```
Comment (rt4914) [inline app/src/main/java/org/oppia/app/options/OptionsReadingTextSizeViewModel.kt]: Fix the sentence to
`/** ReadingTextSize settings view model for the recycler view in [OptionsFragment]. */`

File: app/src/main/java/org/oppia/app/player/exploration/DefaultFontSizeStateListener.kt [hunk 2]
```diff
@@ -1,12 +1,12 @@
 package org.oppia.app.player.exploration
 
-import org.oppia.app.model.StoryTextSize
+import org.oppia.app.model.ReadingTextSize
 
 /**
  * To set the font-size correctly we need to fetch it before loading the `ExplorationFragment` and
  * therefore this listener listens to the default story size in `ExplorationManagerFragment` and
  * passes the information to `ExplorationActivity` which eventually loads the `ExplorationFragment`.
  */
```
Comment (rt4914) [inline app/src/main/java/org/oppia/app/player/exploration/DefaultFontSizeStateListener.kt:9]: Nit:
```
/**
 * To set the font-size correctly we need to fetch it before loading the `ExplorationFragment` and
 * therefore this listener listens to the default reading text size in `ExplorationManagerFragment` and
 * passes the information to `ExplorationActivity` which eventually loads the `ExplorationFragment`.
 */
```

File: app/src/main/java/org/oppia/app/testing/TestFontScaleConfigurationUtilActivity.kt [hunk 3]
```diff
@@ -15,17 +15,17 @@ class TestFontScaleConfigurationUtilActivity : InjectableAppCompatActivity() {
...
-    val storyTextSize = intent.getStringExtra(FONT_SCALE_EXTRA_KEY)
-    configUtilActivityPresenter.handleOnCreate(storyTextSize)
+    val readingTextSize = intent.getStringExtra(FONT_SCALE_EXTRA_KEY)
+    configUtilActivityPresenter.handleOnCreate(readingTextSize)
   }
 
   companion object {
     private const val FONT_SCALE_EXTRA_KEY = "FONT_SCALE_EXTRA_KEY"
 
     /** Returns a new [TestFontScaleConfigurationUtilActivity] for context and story text size. */
```
Comment (rt4914) [inline app/src/main/java/org/oppia/app/testing/TestFontScaleConfigurationUtilActivity.kt]: Update this comment

File: app/src/main/java/org/oppia/app/utility/FontScaleConfigurationUtil.kt [hunk 4]
```diff
@@ -25,12 +25,12 @@ class FontScaleConfigurationUtil @Inject constructor() {
   }
 
   /** This method returns font scale by story text size. */
```
Comment (rt4914) [inline app/src/main/java/org/oppia/app/utility/FontScaleConfigurationUtil.kt]: Update this comment

**Review by rt4914 — COMMENTED** [review]
Great work. Few minor changes suggested. Thanks.

**Comment by prayutsu** [comment]
@rt4914 PTAL

**Comment by prayutsu** [comment]
@rt4914 PTAL

**Comment by prayutsu** [comment]
@anandwana001 what about this one?

**Comment by prayutsu** [comment]
@rt4914 @anandwana001 Please review this PR so that it can be merged before conflicts arise

**Review by rt4914 — APPROVED** [review]
Great. Just update this PR and we can merge this.
Also, please make sure that you always reply to comments and do not "Resolve" them on your own. The reviewer will resolve the comments based on checking the code as well as what you have replied.

**Comment by prayutsu** [comment]
> Great. Just update this PR and we can merge this.
> Also, please make sure that you always reply to comments and do not "Resolve" them on your own. The reviewer will resolve the comments based on checking the code as well as what you have replied.

Sure, I'll take care of that

**Comment by prayutsu** [comment]
> Great. Just update this PR and we can merge this.
> Also, please make sure that you always reply to comments and do not "Resolve" them on your own. The reviewer will resolve the comments based on checking the code as well as what you have replied.

@rt4914 Do I need to create another PR that is up to date with develop branch or that is resolved?

**Comment by rt4914** [comment]
> > Great. Just update this PR and we can merge this.
> > Also, please make sure that you always reply to comments and do not "Resolve" them on your own. The reviewer will resolve the comments based on checking the code as well as what you have replied.
> 
> @rt4914 Do I need to create another PR that is up to date with develop branch or that is resolved?

No no, I have already merged it. You can start working on the new issue that was assigned to you. Thanks.

**Comment by prayutsu** [comment]
> > > Great. Just update this PR and we can merge this.
> > > Also, please make sure that you always reply to comments and do not "Resolve" them on your own. The reviewer will resolve the comments based on checking the code as well as what you have replied.
> > 
> > 
> > @rt4914 Do I need to create another PR that is up to date with develop branch or that is resolved?
> 
> No no, I have already merged it. You can start working on the new issue that was assigned to you. Thanks.

Thanks!

**Pull request merged**